Documentation/contributing/testing/ci.rst

```diff
@@ -221,6 +224,16 @@ For a full list of Jenkins Jobs, see `Jenkins
 <https://jenkins.cilium.io/view/PR/>`_. Trigger phrases are configured within
 each job's build triggers advanced options.
 
+``test-older-k8s`` allows testing of all K8s versions 1.14 through 1.19 (it will trigger the appropriate ``Cilium-PR-K8s-X.XX-kernel-4.9`` jobs).
+If a specific K8s version fails, it can be re-ran using ``test-x.xx-4.9`` (with ``x.xx`` being the specific K8s version).
```

Comment:
Could we have a line about this trigger, or an asterisk, or anything really, in the table above? When I'm simply looking for the trigger phrase for a given job I give a quick look at the array, I don't always read the text below :/.

Reply:
OK!